thirdeye-persistence/src/test/java/ai/startree/thirdeye/datalayer/bao/TestAnomalyManager.java

```diff
@@ -531,4 +532,67 @@ private AnomalyDTO persist(final AnomalyDTO anomaly) {
     assertThat(id).isNotNull();
     return anomaly;
   }
+
+  @Test
+  public void ignoreLabelIndexNoLabelsTest() {
+    final Long id = mergedAnomalyResultDAO.save(buildAnomaly(0, 1));
```

Comment:
just create another function here if you are using `buildAnomaly(0, 1)` and you don't care about the start and end time.

Reply:
done